components/decorateMapComponent.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable */
+
+import { PropTypes } from 'react';
+import {
+  requireNativeComponent,
+  NativeModules,
+  Platform,
+} from 'react-native';
+
+export const SUPPORTED = 'SUPPORTED';
+export const USES_DEFAULT_IMPLEMENTATION = 'USES_DEFAULT_IMPLEMENTATION';
+export const NOT_SUPPORTED = 'NOT_SUPPORTED';
+
+export const PROVIDER_DEFAULT = 'default';
```

Comment:
Let's also export these on `MapView`, so people can do:

```js
<MapView provider={MapView.PROVIDER_GOOGLE}>
  ...
</MapView>
```

Perhaps we should pull these out to a `ProviderConstants.js` or similar.